.github/workflows/weekly.yml

```diff
@@ -30,34 +30,21 @@ jobs:
     strategy:
       matrix:
         flink_branches: [{
-          flink: 1.18-SNAPSHOT,
-          branch: main
-        }, {
           flink: 1.19-SNAPSHOT,
-          jdk: '8, 11, 17, 21',
           branch: main
         }, {
           flink: 1.20-SNAPSHOT,
-          jdk: '8, 11, 17, 21',
           branch: main
         }, {
-          flink: 1.18.1,
+          flink: 1.19.1,
           branch: v3.2
         }, {
-          flink: 1.19.0,
-          branch: v3.2,
-          jdk: '8, 11, 17, 21',
-        }, {
-          flink: 1.18.1,
-          branch: v3.1
-        }, {
-          flink: 1.19.0,
-          branch: v3.1,
-          jdk: '8, 11, 17, 21',
+          flink: 1.20.0,
+          branch: v3.2
```

Comment:
This change feels strange. For an already released connector version `3.2` we are changing the supported Flink version.

We should document this somewhere, why we can do this, and what kind of guarantees we offer.

Reply:
v3.2 is not a released connector but the release branch in case we need to do bugfixes later.